reverse.py

```diff
@@ -0,0 +1,40 @@
+from string import punctuation, digits
+
+def reversed_func(text):
+    """
+    The function reverses the letters in all words of input text.
+    The order of words preserves.
+    All non-letter symbols/numbers stays on the same places.
+    """
+    PRESERVED_CHARS = punctuation + digits
+    reversed_text = []
+
+    for word in text.split():
+        char_list = []
+        reversed_word = ''
+        for char in word:
+            if char not in PRESERVED_CHARS:
+                char_list.append(char)
+
+        for char in word:
```

Comment:
Why just `char`? Don't be shy, it's not like the fragment of code is huge, you can allow yourself to specify the name of the variable in full. We aim to write python code as it was an English novel